From f79ebc5bc696e8747c79a2277ffcf8d91bdadfd5 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 23 Apr 2025 11:52:48 -0700
Subject: [PATCH] openmp: Do not emit date and time into generate files
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This improves reproducibility where the generated files do not
contain time stamps e.g.

//·The·file·was·generated·from·en_US.txt·by·message-converter.py·on·Wed·Apr·23·16:00:53·2025.·//
./usr/src/debug/openmp/20.1.2/runtime/src/kmp_i18n_id.inc

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 openmp/runtime/tools/message-converter.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/openmp/runtime/tools/message-converter.py b/openmp/runtime/tools/message-converter.py
index a493d64c1692..0bd5f929fab6 100644
--- a/openmp/runtime/tools/message-converter.py
+++ b/openmp/runtime/tools/message-converter.py
@@ -188,11 +188,10 @@ def insert_header(f, data, commentChar="//"):
     f.write(
         "{0} Do not edit this file! {0}\n"
         "{0} The file was generated from"
-        " {1} by {2} on {3}. {0}\n\n".format(
+        " {1} by {2} {0}\n\n".format(
             commentChar,
             os.path.basename(data.filename),
             os.path.basename(__file__),
-            datetime.datetime.now().ctime(),
         )
     )
 
